test-data/unit/deps.test

```diff
@@ -432,8 +432,8 @@ def f(x: A) -> None:
         x.y
 [builtins fixtures/isinstancelist.pyi]
 [out]
-<m.<subclass of "A" and "B">.y> -> m.f
-<m.A> -> <m.f>, m.A, m.f
+<m.A.__await__> -> <m.A>
+<m.A> -> <m.f>, m.A, m.f, typing.Awaitable
```

Comment:
whats this

Reply:
When mypy searches for a missing member in a type is checks if it's awaitable, now that we are searching through intersections one by one this will appear if the first entry of the intersection doesn't contain the member we are looking for.